python/ambassador/envoy/v3/v3httpfilter.py

```diff
@@ -262,7 +262,7 @@ def V3HTTPFilter_authv1(auth: IRAuth, v3config: 'V3Config'):
         }
 
     if auth.proto == "grpc":
-        protocol_version = auth.get('protocol_version', 'v2')
+        protocol_version = auth.get('protocol_version', 'v3')
```

Comment:
Do we have checks elsewhere that rule out passing `v2` here?

Reply:
What's up with this change? we still support ext_authz protocol v2.